src/sinks/blackhole.rs

```diff
@@ -70,14 +70,11 @@ impl StreamSink for BlackholeSink {
     async fn run(&mut self, mut input: BoxStream<'_, Event>) -> Result<(), ()> {
         while let Some(event) = input.next().await {
             let message_len = match event {
-                Event::Log(log) => log
-                    .get(crate::config::log_schema().message_key())
-                    .map(|v| v.as_bytes().len())
-                    .unwrap_or(0),
-                Event::Metric(metric) => {
-                    serde_json::to_string(&metric).map(|v| v.len()).unwrap_or(0)
-                }
-            };
+                Event::Log(log) => serde_json::to_string(&log),
+                Event::Metric(metric) => serde_json::to_string(&metric),
```

Comment:
Perhaps there is a better measurement of bytes processed since the internal representation of metrics are much more compact than the expanded form. 🤔 

But, c'est la vie, this seems correct given the situation at hand.

Reply:
Agree, this change is solely for logs. But knowing total bytes for all data is still nice.